src/public/js/vendor/inspectlet.ts

```diff
@@ -0,0 +1,40 @@
+export {};
+const wid = 812310448;
+
+declare global {
+  interface Window {
+    __insp: Array<Array<string | number>>,
+    __inspld: number
+  }
+}
+
+const getKey = async () => {
+  const data = await fetch('/api/wid/');
```

Comment:
It is not clear why do we need to call an API to determine whether inspectlet is enabled or not. That is not desirable and inefficient.

Since the web application initial html is in index.ejs and that is being rendered by server, you can pass down the value in the script tag. Ex:

<script>window.inspectlet.wid = 'xxxx'</script>

`xxxx` is coming from config.

Reply:
Sir, index.ejs is empty, are you talking about layout.ejs here

Reply:
Best to figure out from here by self Himanshu.